### ISSUE TITLE
test_example file

### ISSUE THREAD

**Issue by dys-c** [qinnzou/DeepCrack#24]
i wanna know if the test_example.txt file also needs  both bmg path and png path

**Comment by dys-c** [comment]
But whether it is a training set or a test set, when reading data, it needs two parameters: img and lab
like this：
        img = cv2.imread(item[0])
        lab = cv2.imread(item[1])
when i just put the img path in the txt file，the item[1] is none. how can i solve ?
